### PR TITLE
Added more tests

### PR DESCRIPTION
**Pull request by djallberto**
**JIRA ticket:** [WDP-109](https://github.com/djallberto/presence-analyzer-anowak/tree/WDP-109/tests)

### CODE REVIEW

File: src/presence_analyzer/tests.py [hunk 1]
```diff
@@ -53,6 +53,50 @@ def test_api_users(self):
...
 
+    def test_mean_time_weekday(self):
+        """
+        Test weekday listing.
+        """
+        resp = self.client.get('/api/v1/mean_time_weekday/10')
+        self.assertEqual(resp.status_code, 200)
+        self.assertEqual(resp.content_type, 'application/list')
+        data = json.loads(resp.data)
+        self.assertEqual(len(data), 7)
```
Comment (dpolcyn) [inline src/presence_analyzer/tests.py]: You are already checking this in next assert.

File: src/presence_analyzer/tests.py [hunk 2]
```diff
@@ -53,6 +53,50 @@ def test_api_users(self):
...
+        self.assertListEqual(
+            data[0],
+            [
+                ["Mon", 29934.639175257733],
```
Comment (dpolcyn) [inline src/presence_analyzer/tests.py]: We use ```'``` in python files.

File: src/presence_analyzer/tests.py [hunk 3]
```diff
@@ -53,6 +53,50 @@ def test_api_users(self):
         self.assertEqual(len(data), 2)
         self.assertDictEqual(data[0], {u'user_id': 10, u'name': u'User 10'})
 
+    def test_mean_time_weekday(self):
+        """
+        Test weekday listing.
```
Comment (dpolcyn) [inline src/presence_analyzer/tests.py]: Doesn't tell me anything

File: src/presence_analyzer/tests.py [hunk 4]
```diff
@@ -53,6 +53,50 @@ def test_api_users(self):
...
+            data[0],
+            [
+                ["Weekday", "Presence (s)"],
+                ["Mon", 2903660],
+                ["Tue", 3110829],
+                ["Wed", 3272534],
+                ["Thu", 3213305],
+                ["Fri", 2954610],
+                ["Sat", 0],
+                ["Sun", 0]]
```
Comment (dpolcyn) [inline src/presence_analyzer/tests.py]: Style

File: src/presence_analyzer/tests.py [hunk 5]
```diff
@@ -86,6 +130,55 @@ def test_get_data(self):
...
+        """
+        data = utils.get_data()
+        weekdays = utils.group_by_weekday(data[10])
+        self.assertAlmostEqual(
+            utils.mean(weekdays[0]),
+            29934
+        )
+        self.assertEqual(
+            utils.mean(weekdays[7]),
+            0
```
Comment (dpolcyn) [inline src/presence_analyzer/tests.py]: It can fit in line 178

File: src/presence_analyzer/utils.py [hunk 6]
```diff
@@ -92,7 +92,7 @@ def seconds_since_midnight(time):
 
 def interval(start, end):
     """
-    Calculates inverval in seconds between two datetime.time objects.
+    Calculates interval in seconds between two datetime.time objects.
```
Comment (dpolcyn) [inline src/presence_analyzer/utils.py]: Explain please

File: src/presence_analyzer/tests.py [hunk 7]
```diff
@@ -86,6 +130,55 @@ def test_get_data(self):
...
+        data = utils.get_data()
+        time = data[10][datetime.date(2012, 7, 5)]['start']
+        self.assertEqual(
+            utils.seconds_since_midnight(time),
+            32917
+        )
+
+    def test_interval(self):
+        """
+        Test calculating intervals.
```
Comment (dpolcyn) [inline src/presence_analyzer/tests.py]: Ditto

File: src/presence_analyzer/tests.py [hunk 8]
```diff
@@ -86,6 +130,55 @@ def test_get_data(self):
...
+        start = data[10][datetime.date(2012, 7, 5)]['start']
+        end = data[10][datetime.date(2012, 7, 5)]['end']
+        self.assertEqual(
+            utils.intervals(start, end),
+            32907
+        )
+
+    def test_mean(self):
+        """
+        Test calculating arithmetic mean.
```
Comment (dpolcyn) [inline src/presence_analyzer/tests.py]: Ditto

File: src/presence_analyzer/tests.py [hunk 9]
```diff
@@ -86,6 +130,55 @@ def test_get_data(self):
...
+        data = utils.get_data()
+        weekdays = utils.group_by_weekday(data[10])
+        self.assertEqual(
+            weekdays[0][0],
+            30927
+        )
+
+    def test_seconds_since_midnight(self):
+        """
+        Test calculating correct amount of seconds since midnight
```
Comment (dpolcyn) [inline src/presence_analyzer/tests.py]: Full stop on the end of docstring.

File: src/presence_analyzer/tests.py [hunk 10]
```diff
@@ -86,6 +130,55 @@ def test_get_data(self):
...
+        """
+        data = utils.get_data()
+        time = data[10][datetime.date(2012, 7, 5)]['start']
+        self.assertEqual(
```
Comment (dpolcyn) [inline src/presence_analyzer/tests.py]: Add more cases. Maybe with 0?

File: src/presence_analyzer/tests.py [hunk 11]
```diff
@@ -53,6 +53,50 @@ def test_api_users(self):
         self.assertEqual(len(data), 2)
         self.assertDictEqual(data[0], {u'user_id': 10, u'name': u'User 10'})
 
+    def test_mean_time_weekday(self):
+        """
+        Test weekday listing.
+        """
+        resp = self.client.get('/api/v1/mean_time_weekday/10')
```
Comment (dpolcyn) [inline src/presence_analyzer/tests.py]: U don't have test case that return 404. Are you sure works well?

**Comment by djallberto** [comment]
**JIRA ticket:** [WDP-109](https://kindergarten.stxnext.pl/browse/WDP-109)

**Pull request merged**